src/vulkan-renderer/wrapper/instance.cpp

```diff
@@ -70,15 +71,13 @@ bool Instance::is_extension_supported(const std::string &extension_name) {
 
 Instance::Instance(const std::string &application_name, const std::string &engine_name,
                    const std::uint32_t application_version, const std::uint32_t engine_version,
-                   const std::uint32_t vk_api_version, bool enable_validation_layers, bool enable_renderdoc_layer,
+                   bool enable_validation_layers, bool enable_renderdoc_layer,
                    const std::vector<std::string> &requested_instance_extensions,
-                   const std::vector<std::string> &requested_instance_layers)
-    : m_vk_api_version(vk_api_version) {
+                   const std::vector<std::string> &requested_instance_layers) {
     assert(!application_name.empty());
     assert(!engine_name.empty());
 
-    // In Vulkan API we can use VK_MAKE_VERSION() macro to create an std::uint32_t value from major, minor and patch
-    // number.
+    constexpr std::uint32_t REQUIRED_VK_API_VERSION = VK_API_VERSION_1_2;
```

Comment:
I bet we should place such a important variable to a more common place not hidden here. Or is it only be used here, then it is ok.
Also it should be in a header.

Reply:
`inline`? Why not add it as a member `static constexpr std::uint32_t REQUIRED_VK_API_VERSION = VK_API_VERSION_1_2;`?

Reply:
That could be also an option.

Reply:
Done